packages/indexer-common/src/network.ts

```diff
@@ -347,11 +347,11 @@ export class Network {
             ) {
               if (await this.transactionManager.isOperator.value()) {
                 logger.info(`Indexer already registered, operator status already granted`)
-                return
               } else {
                 logger.info(`Indexer already registered, operator status not yet granted`)
               }
```

Comment:
Shouldn't the return go here, so that if the url/geohash doesn't match, it continues with registration?

Reply:
Or under line 352?  Should avoid sending the registration tx if: already registered, url unchanged, and geoHash unchanged regardless of the operator status.

Reply:
Ah yeah that's what I meant, should've commented one line below 👍